src/main/java/com/tsystems/sbs/DefaultRegexpPairs.java

```diff
@@ -19,7 +19,29 @@ public final class DefaultRegexpPairs {
                     new RegexpPair("password=\\S*", "password=********") //PASSWORD MASKING
             ));
 
+    private final static List<RegexpPair> DEFAULT_REGEXES_AWS
+            = Collections.<RegexpPair>unmodifiableList(
+            Arrays.<RegexpPair>asList(
+                    new RegexpPair("(AWS_[a-zA-Z_]+=)(\\S+)", "$1********"), // AWS RegExp MASKING
+                    new RegexpPair("(aws_[a-zA-Z_]+=)(\\S+)", "$1********")
+            ));
+
+    private final static List<RegexpPair> DEFAULT_REGEXES_DD
+            = Collections.<RegexpPair>unmodifiableList(
```

Comment:
This is very generic any 32/40 char long word will be replaced, this does not feel good. Can we do better?

Reply:
Agree Ill have a look.